app/components/settings/faq/index.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import {
+  ScrollView
+} from 'react-native';
+// import tracker from 'helpers/googleAnalytics';
+
+import I18n from 'locales';
+import Theme from 'config/theme';
+import List from 'components/common/list';
+import styles from './styles';
+
+class FaqList extends Component {
+  static navigatorStyle = {
+    navBarTextColor: Theme.colors.color1,
+    navBarButtonColor: Theme.colors.color1,
+    topBarElevationShadowEnabled: false,
+    navBarBackgroundColor: Theme.background.main
+  };
+
+  static propTypes = {
+    navigator: PropTypes.object.isRequired
+  };
+
+  componentDidMount() {
+    // tracker.trackScreenView('TermsAndConditions');
```

Comment:
I guess that we could remove the tracker from FAQ-detail and only track this one with FAQ